core/blockchain_action.go

```diff
@@ -156,11 +156,6 @@ func (chain *FullBlockChain) verifyTxs(bh *types.BlockHeader, txs []*types.Trans
 		return -1
 	}
 
-	if !chain.validateTxRoot(bh.TxTree, txs) {
```

Comment:
This check should not be removed. need to validate the transactions root hash in a block

Reply:
ok